tezster-manager.js

```diff
@@ -34,8 +34,8 @@ class TezsterManager{
         this.setup.getLogFiles();
     }
 
-    setProvider(){
-        this.accounts.setProvider(args);
+    setProvider(accountSetProviderParameterValues){
```

Comment:
why are you combinign multiple items in a single CL? 
it should be done in a separate PR.

Reply:
I will take care of this further.